client/src/components/LocationMap.tsx

```diff
@@ -90,16 +110,18 @@ export function LocationPickerMap( props: { setLocation:  (location: string, lat
       </FormGroup>
       <Paper
         elevation={5}
-        style={{ height: showMap ? '35vh' : '0', width: '100%' }}
+        style={{ height: showMap ? '35vh' : '0', width: '100%' }} // don't unload the map, 0 height will hide it
```

Comment:
usually we want to unload and not hide, save some resources.

Reply:
Can't unload it, the component loads the google maps API, unloading it also unloads the API.